CMakeLists.txt

```diff
@@ -33,6 +35,10 @@ endif()
 
 add_definitions ( -D_VERSION=${CMAKE_PROJECT_VERSION} -D_PACKAGE=${CMAKE_PROJECT_NAME} -DSYSCONFDIR="${CMAKE_INSTALL_FULL_SYSCONFDIR}")
 
+if(CURSES_OLD_KEYS)
```

Comment:
This check is backwards. Also, we could scope it to just the curses tn5250 binary with `target_compile_definitions`.